Cairo Desktop/CairoDesktop.Configuration/Settings.cs

```diff
@@ -108,6 +109,23 @@ public static bool IsFirstRun
             }
         }
 
+        public static bool RunAtLogOn
```

Comment:
Since this shouldn't be a stored setting, references should be removed from this file as well.